File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file105.f90
program DRB102_copyprivate_orig_no
    use omp_lib
    use DRB102
    implicit none

    !$omp parallel
        !$omp single
        x=1.0
        y=1
        !$omp end single copyprivate(x,y)
    !$omp end parallel

    print 100, x, y
    100 format ('x =',F3.1,2x,'y =',i3)

end program

File: data/gpt-4o/readable_format_HPEC/cpp/file105.cpp
#include <stdio.h>
#include <omp.h>

double x;
int y;

int main() {
#pragma omp parallel
    {
#pragma omp single copyprivate(x, y)
        {
            x = 1.0;
            y = 1;
        }
    }
    printf("x = %.1f  y = %d
", x, y);
    return 0;
}
